app/models/verification/residence.rb

```diff
@@ -65,6 +66,10 @@ def store_failed_attempt
     })
   end
 
+  def adult?
+    (Date.today.year - self.date_of_birth.year) > 18
```

Comment:
hmmm I see your point, let me fix it. So.. I should check the javascript part as well.